tools/erc-repository-indexer/erc-contract-indexer/README.md

```diff
@@ -35,11 +35,12 @@ The ERC Contract Indexer is a tool designed to facilitate the indexing and manag
-| ----------------- | ----------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| `HEDERA_NETWORK`  | The network to connect to.                | `previewnet`, `testnet`, or `mainnet`                                                                                                                                                                                           |
-| `MIRROR_NODE_URL` | The URL for the Hedera mirror node API.   | A valid URL pointing to the Hedera mirror node (e.g., `https://{previewnet\|testnet\|mainnet}.mirrornode.hedera.com`)                                                                                                           |
-| `STARTING_POINT`  | The starting point for contract indexing. | A Hedera contract ID (e.g., `0.0.369`), an EVM 20-byte address (e.g., `0x0000000000000000000000000000000000000369`), or a get contract list next pointer (e.g., `/api/v1/contracts?limit=100&order=asc&contract.id=gt:0.0.369`) |
+| Variable               | Description                                                                                                  | Accepted Values                                                                                                                                                                                                                 |
+| ---------------------- | ------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `HEDERA_NETWORK`       | The network to connect to.                                                                                   | `localnet`, `previewnet`, `testnet`, or `mainnet`                                                                                                                                                                               |
+| `MIRROR_NODE_URL`      | The URL for the Hedera mirror node API.                                                                      | A valid URL pointing to the Hedera mirror node (e.g., `https://{previewnet\|testnet\|mainnet}.mirrornode.hedera.com`)                                                                                                           |
+| `MIRROR_NODE_URL_WEB3` | The URL for the Hedera Mirror Node Web3Module API, required only when `HEDERA_NETWORK` is set to `localnet`. | Any value                                                                                                                                                                                                                       |
```

Comment:
Does it make sense to have these as command line arguments as well? This is to avoid users to create a `.env`

Reply:
Ah yeah nice one but prob gonna stick to .env for now.

Reply:
Yeap, sounds good, thanks.